packages/portal-proto/src/components/SearchInput.tsx

```diff
@@ -227,14 +385,14 @@ export const SearchInput: React.FC = () => {
                             tooltip: "text-black drop-shadow-md rounded-none",
                           }}
                           offset={17}
+                          opened={activedescendant === index ? true : undefined}
```

Comment:
Could this just be `opened={activedescendant === index}`?

Reply:
added comment to explain why its this way